.github/workflows/test-all-packages.yml

```diff
@@ -533,7 +533,7 @@ jobs:
       - name: docker build (sdk)
         run: cd packages/deployment && ./scripts/test-docker-build.sh | $TEST_COLLECT
       - name: docker build upgrade test
-        run: cd packages/deployment/upgrade-test && docker build --build-arg BOOTSTRAP_MODE=${{ matrix.bootstrap-version }} -t docker-upgrade-test:latest -f Dockerfile upgrade-test-scripts
+        run: cd packages/deployment/upgrade-test && docker build --build-arg BOOTSTRAP_MODE=${{ matrix.bootstrap-version }} --build-arg DEST_IMAGE=ghcr.io/agoric/agoric-sdk:latest -t docker-upgrade-test:latest -f Dockerfile upgrade-test-scripts
```

Comment:
should this be dev?

Reply:
I think it shouldn't, otherwise we're running against a different agoric-sdk than the local source tree. I believe as is it builds a local image and runs against that.

Reply:
It may be dev actually -- are we certain that at this point we have built and tagged `dev` (just not uploaded it yet)?

Reply:
Looking a bit closer, I think it should be `latest`, because we want to use the image produced by the previous `make docker-build-sdk` command so tests run against the local SDK. If we point it at dev we'll run against the latest published `dev` image that might be different. The `dev` images are only tagged on merge to `master`.